week3/homework/src/index.js

```diff
@@ -1,3 +1,50 @@
-'use strict';
+const express = require('express');
+const app = express();
 
-// TODO: Write the homework code in this file
+const bodyParser = require('body-parser');
+const getList = require('./getList');
+const getToDo = require('./getTodo');
+const deleteAll = require('./deleteAll');
+const deleteOne = require('./deleteOne');
+const addOne = require('./addToDo');
+const markDone = require('./markDone');
+const editToDo = require('./editToDo');
+
+app.use(express.json());
+app.use(bodyParser.json());
+
+app.post('/addTodo', (req, res) => {
```

Comment:
In a restful API is the url is a noun and the http method is the verb
todo = noun
post = verb = create

The idea is precisely to avoid urls such as addTodo/createTodo/newTodod (mainly to avoid confusion, as you can see there are many way how to name a thing as simple as creating a new todo)

Reply:
i'll work on that hopefully it's better now